### PR TITLE
Add delay time to config Conflagrate

### PR DESCRIPTION
**Pull request by Motorheadx**
##### CHANGES PROPOSED:
Add delay time to config Conflagrate

- Closes
https://github.com/azerothcore/azerothcore-wotlk/issues/2302

##### TESTS:
No problem

##### HOW TO TEST THE CHANGES:
Create warlock and cast a spell Conflagrate

### CODE REVIEW

**Comment by Stoabrogga** [comment]
Why do we need a worldserver config parameter to change the behaviour of single spells? Sounds strange to me.

**Comment by Motorheadx** [comment]
So that everyone can set the delay for Conflagrate at will and not rebuild the server many times

File: src/server/worldserver/worldserver.conf.dist [hunk 1]
```diff
@@ -3276,5 +3276,12 @@ Calculate.Gameoject.Zone.Area.Data = 0
 
 LFG.Location.All = 0
 
+#
+#     DelayConflagrate
+#        Description: Set time delay сonflagrate
+#        Default:     700 ms
+
+DelayConflagrate = 50
```
Comment (FrancescoBorzi) [inline src/server/worldserver/worldserver.conf.dist]: the default should be 700

**Comment by FrancescoBorzi** [comment]
We can't afford adding custom configuration for each spell, because there are just too many.

Is there any specific reason to have this?

**Comment by Stoabrogga** [comment]
It would also be great to explain what's the actual issue with Conflagrate. Neither in this PR nor in the associated issue there's an adequate description.

**Comment by Motorheadx** [comment]
I understand you good luck

**Pull request closed without merging**